client/src/components/layout/DeleteBuildButton.js

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+
+const DeleteBuildButton = ({currentUserData, dataToDelete, setUserData}) => {
+    
+    const buildId = dataToDelete
+    const deleteBuild = async () => {
+        const responseFromDeletion = await fetch(`/api/v1/builds/${buildId}`, {
+            method: "DELETE",
+            headers: new Headers({ "Content-Type": "application/json" })
+        })
+        const parsedResponse = await responseFromDeletion.json()
+        console.log("Parsed Response: ", parsedResponse)
+        if(parsedResponse.deleted){
```

Comment:
fix the spacing for this if statement
https://github.com/airbnb/javascript?tab=readme-ov-file#whitespace--around-keywords
https://github.com/airbnb/javascript?tab=readme-ov-file#whitespace--before-blocks